.github/instructions/Data.instructions.md

```diff
@@ -0,0 +1,27 @@
+---
+applyTo: "Source/Data/*"
+---
+
+These files define the mapping of phrases and words to their Bopomofo representations.
+
+In most cases, developers will add new Chinese characters or phrases rather than delete existing ones. When adding new characters or phrases, please perform the following checks:
+
+## Adding Phrases 
+
+When adding a new phrase containing multiple characters to `BPMFMappings.txt`, ensure that `phrase.occ` is also updated with the same phrase and its frequency. The frequency should be a positive integer.
```

Comment:
nit: the frequency may be 0. Perhaps rephrase as `The frequency should be zero or a positive integer.`?